appengine/test/tc_base_image.rb

```diff
@@ -0,0 +1,18 @@
+require "minitest/autorun"
```

Comment:
I'd rather keep it here. I prefer to be able to run test cases independent of rake. (Unchanged in https://github.com/GoogleCloudPlatform/ruby-docker/pull/24)